src/module/actor/character/sheet.ts

```diff
@@ -315,25 +315,48 @@ class CharacterSheetPF2e<TActor extends CharacterPF2e> extends CreatureSheetPF2e
 
         sheetData.abpEnabled = AutomaticBonusProgression.isEnabled(actor);
 
+        // get the backing language for common
+        const commonLanguage = game.settings.get("pf2e", "homebrew.languageRarities").commonLanguage;
+
         sheetData.languages = ((): LanguageSheetData[] => {
             const languagesBuild = actor.system.build.languages;
             const sourceLanguages = actor._source.system.details.languages.value.filter(
                 (l) => l in CONFIG.PF2E.languages,
             );
-            const isOverMax = languagesBuild.value > languagesBuild.max;
+
+            const deductBackingLanguage = commonLanguage
```

Comment:
This is (or should be) guaranteed to be present

Reply:
Even if its not, the following logic will still work out fine, the actor wouldn't have a common language of null.

Reply:
Bad variable name there. I refactored it - it's supposed to be the backing language.